ui/src/utils/constants.js

```diff
@@ -31,7 +31,7 @@ export const VIEW_ROLES_PAGE_SIZE = 1000;
 export const VIEW_SECRETS_PAGE_SIZE = 100;
 export const VIEW_DENYLIST_PAGE_SIZE = 20;
 export const VIEW_NAMESPACES_PAGE_SIZE = 20;
-export const VIEW_CACHE_PURGES_PAGE_SIZE = 20;
+export const VIEW_CACHE_PURGES_PAGE_SIZE = 1000;
```

Comment:
👌

Reply:
Thanks. Hey @helfi92, any plans on migrating class-based components into functional hook components in the future. Just asking :)

Reply:
For existent components, we can keep the current behavior to avoid introducing new bugs but for new components, we can definitely use hook components.

Reply:
Fair point @helfi92